.eslintrc.js

```diff
@@ -826,6 +826,17 @@ module.exports = {
           {
             // prevents UI code from importing server side code and then webpack including it when doing builds
             patterns: ['**/server/*'],
+            paths: [
+              /*
+                prevents importing raw useSelector which is using different equality function than mapStateToProps,
+                so to make sure we keep the logic while moving to hooks let's use useShallowEqualSelector
+              */
+              {
+                name: 'react-redux',
+                importNames: ['useSelector'],
+                message: 'Please use "useShallowEqualSelector" instead or create your own selector',
```

Comment:
you'll still get the error when you create your own selector correct?

Reply:
correct

Reply:
I don't think we should add a rule here. Many teams work in this code base, for that reason I think we should all use the exact set of lint rules defined by Kibana core. 

The Resolver code base, for example, has no reason to use shallow selectors.